### DISCUSSION TITLE
Unable to install shell completions in ZSH

### DISCUSSION THREAD

**Discussion by Wulfheart** [casey/just#2125] [Q&A]
Hello,

I tried several options and I think that this is a skill issue but I would greatly appreciate any assistance you can offer. I added it (analogue to my `eval "$(atuin init zsh)"`) with `eval "$(just --completions zsh)"` to my `~/.zshrc` and I get the error message "_arguments:comparguments:325: can only be called from completion function". How can I install the autocompletions?

**Comment by flyingicedragon** [comment]
`just --completions zsh` is used for generateing a completion script. I used `just --completions zsh > ~/.oh-my-zsh/completions/_just.zsh`.
I am a `oh-my-zsh` user. `oh-my-zsh` had added this path to `$fpath` and loaded it.

I guess `autoload -U compaudit compinit` will also work if you don't use `oh-my-zsh` but I never try.
